usaspending_api/references/v2/views/filter_trees/filter_tree.py

```diff
@@ -0,0 +1,63 @@
+from abc import ABCMeta, abstractmethod
+from dataclasses import dataclass
+
+DEFAULT_CHILDREN = 0
+
+
+@dataclass
+class Node:
+    id: str
+    ancestors: list
+    description: str
+    count: int
+    children: list
+
+    def toJSON(self):
```

Comment:
camel case? in python code?

Reply:
seconded

Reply:
This method isn't returning JSON

Reply:
Would you consider "to_dictionary" more accurate?

Reply:
Yes. Would you?

Reply:
Technically, yes, but the way people use the term colloquially it's kind of like the difference between url and uri. The Response class we use has a "json()" method that returns a dictionary. Not a hill to die on though.

Reply:
I was amused people jumped on the camel case when the name was inaccurate. The example is different since the Request object contains JSON and provides that data in a native datatype. Keep `to_json()` if it works for the devs actually reviewing.

Reply:
language-specific casing is like unused import statements; we all know it's not important, but it's really obvious, and I feels good to make a comment knowing you can't be wrong.